inc/plugins/hello.php

```diff
@@ -83,6 +83,27 @@ function hello_info()
 	);
 }
 
+/*
+ * _configure():
+ *    Called to determine the configure link shown in the plugins list. This is only called when
+ *    the plugin is installed. Return `false` to prevent the configure link from being displayed.
+*/
+function hello_configure()
+{
+	global $db;
+
+	$query = $db->simple_select('settinggroups', 'gid', "name='hello'");
+
+	$gid = (int)$db->fetch_field($query, 'gid');
+
+	if($gid > 0)
+	{
+		return "index.php?module=config-settings&amp;action=change&amp;gid={$gid}";
```

Comment:
Is returning the url the best option? Future routing improvements may change url structure. 
IMHO returned `GID` gives us more freedom.

Reply:
I'd agree with that @Matslom. Returning GID seems much more useful to me.

Reply:
I went with URL as many plugins use a new module to hold their configuration. Only accepting a GID as the return would prevent the plugin from linking to the module.